### PR TITLE
Specify that email handling converts to lowercase first

### PR DESCRIPTION
**Pull request by turt2live**
Spec for https://github.com/matrix-org/matrix-doc/pull/2265

### CODE REVIEW

**Review by uhoreg — APPROVED** [review]
typo, but otherwise lgtm

File: content/appendices.md [hunk 1]
```diff
@@ -772,6 +772,13 @@ Represents E-Mail addresses. The `address` is the raw email address in
 `user@domain` form with the domain in lowercase. It must not contain
 other text such as real name, angle brackets or a mailto: prefix.
 
+In addition to lowercasing the domain component of an email address,
+implementations are expected to lowercase the localpart as described
```
Comment (richvdh) [inline content/appendices.md:776]: lowercase != casefold.

Comment (turt2live) [reply content/appendices.md:776]: Is this a request to fix it?

Comment (richvdh) [reply content/appendices.md:776]: yes please!

Comment (turt2live) [reply content/appendices.md:776]: https://github.com/matrix-org/matrix-doc/issues/3175

**Pull request merged**